src/server.js

```diff
@@ -1,14 +1,26 @@
 //Create a basic express server
 const express = require("express");
 const app = express();
+const mongoose = require("mongoose");
+
 require("dotenv").config();
+
 const port = process.env.SERVER_PORT || 3000;
+const db = process.env.DB_URL || "mongodb://localhost:27017/tyta";
```

Comment:
Do we really put a default URL? Or we can create a check for users' environment configuration?